proprietary/design-tokens/figma/figma.tokens.json

```diff
@@ -3134,6 +3134,23 @@
       }
     }
   },
+  "components/logo": {
+    "utrecht": {
+      "logo": {
+        "max-block-size": {},
+        "max-inline-size": {},
+        "min-block-size": {},
+        "min-inline-size": {}
+      }
+    }
+  },
+  "components/logo-text": {
+    "frameless": {
+      "logo-text": {
+        "font-family": { "value": "{frameless.typography.font-family.secondary}" }
+      }
+    }
```

Comment:
Omdat "logo" een _block_ is en "text" een _element is, moeten ze genest worden. Dan wordt de design token `logo.text`. Nu ziet het eruit alsof `logo-text` een zelfstandig component is.

```suggestion
  "components/logo-text": {
    "frameless": {
      "logo": {
        "text": {
          "font-family": { "value": "{frameless.typography.font-family.secondary}" }
        }
      }
    }
```

Reply:
Misschien dat ik je verkeerd begrijp maar LogoText is een component toch? 🤔 

<img width="317" alt="Scherm­afbeelding 2024-05-28 om 10 43 32" src="https://github.com/frameless/website/assets/112861180/ba67f9d8-2b7b-4cfe-90a3-1448f8834726">

Reply:
✅